frontend/www/js/omegaup/components/arena/RunsForCourses.vue

```diff
@@ -243,215 +243,223 @@
           </tfoot>
           <tbody>
             <tr v-for="run in paginatedRuns" :key="run.guid">
-              <td>{{ time.formatDateLocalHHMM(run.time) }}</td>
-              <td v-show="showGUID">
-                <acronym :title="run.guid" data-run-guid>
-                  <tt>{{ getShortGuid(run.guid) }}</tt>
-                </acronym>
+              <td v-if="run.status !== 'ready'" colspan="8" data-new-run>
```

Comment:
The attribute `data-new-run` corresponds to the new-submit button, and here you are displaying the SkeletonLoader. So it can be removed:
```suggestion
              <td v-if="run.status !== 'ready'" colspan="8">
```